src/Lykke.Service.Decred.Api.Services/HealthService.cs

```diff
@@ -66,11 +83,11 @@ private async Task<HealthIssue[]> GetDcrdHealthIssues()
             }
             catch (Exception e)
             {
-                await _log.WriteErrorAsync(nameof(HealthService), nameof(GetDcrdHealthIssues), "", e);
+                _log.Error(e,process: nameof(GetDcrdHealthIssues));
```

Comment:
Containing method name will passed to the process parameter automatically by default, it's not needed to pass it manually